drivers/sensor/renesas/rrh46410/rrh46410.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_RRH46410_RRH46410_H_
+#define ZEPHYR_DRIVERS_SENSOR_RRH46410_RRH46410_H_
+
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/rtio/rtio.h>
+
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
+#include <zephyr/drivers/i2c.h>
+#endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c) */
+
+#define RRH46410_OPERATION_MODE_IAQ_2ND_GEN	0x01
+#define RRH46410_GET_OPERATION_MODE			0x10
+#define RRH46410_SET_OPERATION_MODE			0x11
+#define RRH46410_SET_HUMIDITY				0x12
+#define RRH46410_GET_MEASUREMENT_RESULTS	0x18
+
+/* 1 byte status, 1 byte sample counter, 8 bytes data, 1 byte checksum */
+#define RRH46410_BUFFER_LENGTH 11
+
+#define RRH46410_MAX_RESPONSE_DELAY 150 /* Add margin to the specified 50 in datasheet */
+
+union rrh46410_bus_cfg {
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
```

Comment:
Please include `#define DT_DRV_COMPAT` in this file otherwise you're depending on include order.